### DISCUSSION TITLE
kpanda/user-guide/backup/etcd-backup

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4411] [Announcements]
# kpanda/user-guide/backup/etcd-backup

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/backup/etcd-backup.html

**Comment by panpan0000** [comment]
备份恢复的最佳实践： https://docs.daocloud.io/kpanda/best-practice/etcd-backup.html?h=etcd

**Reply by windsonsea** [reply]
Added this reference to best practice in PR #5303
